src/components/pages/confirm/membership/index.tsx

```diff
@@ -185,28 +187,64 @@ export const ConfirmMembership: React.FC<ConfirmMembershipProps> = ({
         </>
       ) : (
         <>
+          {currentState.matches('pending') && <p>Setting up your account...</p>}
```

Comment:
Requesting ideas for how to style and word this.

The idea is that while the client polls in the background for an auth token from the server, this page will display a message explaining that we are in the process of setting up their account.